docs/gl_objects/pipelines_and_jobs.rst

```diff
@@ -33,9 +33,9 @@ Get variables of a pipeline::
 
     variables = pipeline.variables.list()
 
-Create a pipeline for a particular reference::
+Create a pipeline for a particular reference with custom variables::
 
-    pipeline = project.pipelines.create({'ref': 'master'})
+    pipeline = project.pipelines.create({'ref': 'master', "variables": [{"key": "MY_VARIABLE", "value": "hello"}]})
```

Comment:
Could we keep this aligned? You've mixed `'` and `"`.

Reply:
fixed